package.json

```diff
@@ -31,9 +31,6 @@
     "url": "https://github.com/open-feature/js-sdk/issues"
   },
   "homepage": "https://github.com/open-feature/js-sdk#readme",
-  "engines": {
-    "node": ">=18"
-  },
```

Comment:
Pretty sure this is doing nothing since it's at the root.

Reply:
yea this is the client side SDK, so yea I don't think this does anything.